### DISCUSSION TITLE
permissions/restrictions on nested directory structure without an ACL for every subdirectory.

### DISCUSSION THREAD

**Discussion by matthewlenz** [drakkan/sftpgo#1058] [Q&A] [answered]
I've got a situation where I've got an /incoming folder and inside that folder I have a couple dozen subfolders.  I only want the user to be able to upload files into the subfolders.  Since there are so many folders I have them being auto-created via an event for new users created with a specific admin account.  the group these users are created with has a basic ACL on the root, and /incoming folders to at least prevent new directories, etc.  Is there anyway to make it so users only have permissions on /incoming/*/ without naming every single subfolder?  I see the per directory pattern restrictions but I'm not sure what it's used for.

**Comment by drakkan** [comment, marked as answer]
https://github.com/drakkan/sftpgo/commit/2017cb60e90c25aadf0a5eb9a04d2de8a38ac56b
